### DISCUSSION TITLE
Section structure ideas

### DISCUSSION THREAD

**Discussion by jasdumas** [r-devel/cran-cookbook#10] [Ideas]
## Background

As we get started on these first few issues of the cookbook, I'm a little stumped on how to organize our content. 

The first skeleton draft had `.qmd` files for **common**, **occasional** and **infrequent** issues to correspond with the sidebar section labels. The current set of identified issues that @Baltmann01 put together has a different grouping of:
- Manuals & Documentation issues
- Code issues
- Description file issues 

## Discussion question

1. Should we first label each of the [issues](https://github.com/orgs/r-devel/projects/3/views/1) by frequency of occurrence to determine which section they might fall into?

1. Should we add just a general issues/best practice section? (I think we might have discussed this in our last team meeting 🤔 )

**Comment by bettinagruen** [comment]
I like the grouping proposed by @Baltmann01 better because it is content driven. The frequency might also change over time if documentation is improved for some issues.

**Reply by jasdumas** [reply]
@bettinagruen I agree! I think our overall objective is to document the most common issues anyway.
